src/sif/truckcost.cc

```diff
@@ -665,6 +675,11 @@ void ParseTruckCostOptions(const rapidjson::Document& doc,
     pbf_costing_options->set_use_tolls(
         kUseTollsRange(rapidjson::get_optional<float>(*json_costing_options, "/use_tolls")
                            .get_value_or(kDefaultUseTolls)));
+
+    // top_speed
+    pbf_costing_options->set_top_speed(
```

Comment:
we should do this in dynamiccost for all the costings (slowly working towards one costing to rule them all)

Reply:
we dont actually have to use it for all costings but if we do it in dynamic cost we can (and eventually should) have it work for all costings

Reply:
yep, makes sense